recipes/simplification/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "simplification" %}
+{% set version = "0.5.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/urschrei/simplification/archive/v{{ version }}.tar.gz
+  sha256: b056fa0ca0cd05f9f7c2303f382d511a2c1f3c116511a097d9f223c950ad180d
+
+build:
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
+  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
+  skip: True  # [py<36]
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python >=3.6
+    - pip >=8.1.2
+  run:
+    - python >=3.6
```

Comment:
Since you're building compiled packages, you shouldn't actually need to put in the Python version specification here and under `host`.

Reply:
removing.